app/jobs/assignment_repo/porter_status_job.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class AssignmentRepo
+  class PorterStatusJob < ApplicationJob
+    class ImportInProgress < StandardError; end
+    queue_as :porter_status
+    retry_on Octopoller::TimeoutError, queue: :porter_status
+
+    # rubocop:disable MethodLength
+    def perform(assignment_repo, user)
+      github_repository = assignment_repo.github_repository
+
+      result = Octopoller.poll do # TODO: log errors
+        begin
+          progress = github_repository.import_progress[:status]
+          case progress
+          when GitHubRepository::IMPORT_COMPLETE
+            Creator::REPOSITORY_CREATION_COMPLETE
+          when *GitHubRepository::IMPORT_ONGOING
```

Comment:
Is this `*` supposed to be here?

Reply:
The `*` lets the `case`  [match against arrays](https://bengribaudo.com/blog/2013/12/31/2530/ruby-case-testing-against-arrays-of-values)